uptane-whitepaper.md

```diff
@@ -71,9 +71,9 @@ The second change made to the basic TUF design has to do with the way Uptane ver
+The diagram above illustrates how the checks and balances of the Uptane system works. The connected components on the right hand side of the diagram are on the vehicle, while the components on the left hand side represent the repositories. The Image repository can be thought of as an authoritative source of information about software images, and the keeper of every image currently deployed by the OEM or supplier, along with the metadata files that prove their authenticity. The Director repository determines what software updates should be distributed to each ECU. 
 
-In the first step in the software update process, the vehicle sends its vehicle version manifest for all installed ECUs to the Director repository. This vehicle version manifest contains signed information about existing, installed software images. Using this input, the Director chooses which software images should be installed next. The metadata and software images are distributed to the vehicle, which then runs a verification process. The diagram above shows a Primary ECU that is connected to a number of Secondary ECUs. The Primary ECU downloads software images and metadata from the Director and Image repositories, and then shares them with one or more Secondary ECUs on the same vehicle. ECUs are classified in terms of their access to storage space, memory, a power supply, and (optionally) a direct Internet connection. The chosen form of verification—*full* or *partial*—is also based on the available resources of the ECU, as well as the criticality of the software update (for safety and/or security). If the verification process finds no issues, then the software image can be flashed to the ECU, and the vehicle version manifest is updated.
+In the first step in the software update process, the vehicle sends its vehicle version manifest for all installed ECUs to the Director repository. This vehicle version manifest contains signed information about existing, installed software images. Using this input, the Director chooses which software images should be installed next. The metadata and software images are distributed to the vehicle, which then runs a verification process. The diagram above shows a Primary ECU that is connected to a number of Secondary ECUs. The Primary ECU downloads software images and metadata from the Director and Image repositories, and then shares them with one or more Secondary ECUs on the same vehicle. ECUs are classified in terms of their access to storage space, memory, a power supply, and (optionally) a direct Internet connection. The chosen form of verification--*full* or *partial*--is based on the available resources of the ECU, as well as the criticality of the software update (for safety and/or security). If the verification process finds no issues, then the software image can be flashed to the ECU, and the vehicle version manifest is updated.
```

Comment:
this hunk replaces an emdash (–) with two dashes (--) which seems to increase consistency of the formatting.